android/src/main/java/com/reactlibrary/RNPurchasesMath.java

```diff
@@ -0,0 +1,39 @@
+package com.reactlibrary;
```

Comment:
(Sorry if I'm missing something super obvious, I have super limited RN knowledge.)

Is there any reason why this is in `com.reactlibrary`?

Reply:
I don't want to expose them. `package com.reactlibrary;` is the package the React Native library needs to be. I could just put them inside a package `utils` because the name is not predictible though. What do you think?

Reply:
Makes sense, I didn't realize it's a common pattern to put RN modules in this package. It's prefixed (kind of funny to see in Java haha), so I think it should be fine.